### PR TITLE
style(schedule): update color and layout for 2022 design

### LINKED ISSUE #294: [Feature Request] Update the color of schedule page
Update the color of the components displayed on the schedule page

image

## reference

- [figma](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=0%3A1)

### PR DESCRIPTION
**Pull request by mattwang44**
resolve #294

##### pc view:

https://user-images.githubusercontent.com/24987826/181892925-9ba82237-05f6-45d8-8774-ca9fb97acd21.mp4

##### mobile view:

image

### CODE REVIEW

File: components/schedule/ScheduleBlock.vue [hunk 1]
```diff
@@ -46,6 +47,7 @@ export default {
                 '-primary': this.primary,
                 '-secondary': this.secondary,
                 '-tertiary': this.tertiary,
+                '-tab-header': this.tabHeader,
                 '-active': this.active,
                 '-mini': this.mini,
                 '-transparent': this.transparent,
```
Comment (furecool) [inline components/schedule/ScheduleBlock.vue:53]: Should it be `'cursor-pointer': this.tabHeader,` ?

Comment (furecool) [reply components/schedule/ScheduleBlock.vue:53]: Means line 55.

Comment (mattwang44) [reply components/schedule/ScheduleBlock.vue:53]: nice catch!
should be true when `this.primary` or `this.tabHeader` is true.

**Review by furecool — APPROVED** [review]
LGTM

**Pull request merged**